storage/gcs.go

```diff
@@ -51,10 +53,14 @@ func (s *GCS) open() (err error) {
 	} else if len(credentialsFile) != 0 {
 		opt = option.WithCredentialsFile(credentialsFile)
 	} else {
-		return fmt.Errorf("GCS: credentials or credentialsFile is required")
+		creds, err := google.FindDefaultCredentials(ctx, storage.ScopeReadOnly)
```

Comment:
Please add a comment to explain here and add a reference link to Google Cloud Docs here.

Reply:
Added comments in code. Also updated to storage.ScopeReadWrite for correctness.